iis/README.md

```diff
@@ -1,32 +1,83 @@
-# Iis Integration
+# IIS Integration
 
-## Overview
+# Overview
 
-Get metrics from iis service in real time to:
+Collect IIS metrics aggregated across all of your sites, or on a per-site basis. The IIS Agent check collects metrics for active connections, bytes sent and received, request count by HTTP method, and more. It also sends a service check for each site, letting you know whether it's up or down.
 
-* Visualize and monitor iis states
-* Be notified about iis failovers and events.
+# Installation
 
-## Installation
+The IIS check is packaged with the Agent, so simply [install the Agent](https://app.datadoghq.com/account/settings#agent) on your IIS servers.
 
-Install the `dd-check-iis` package manually or with your favorite configuration manager
+Also, your IIS servers must have the `Win32_PerfFormattedData_W3SVC_WebService` WMI class installed. 
 
-## Configuration
+# Configuration
 
-Edit the `iis.yaml` file to point to your server and port, set the masters to monitor
+## Prepare IIS
 
-## Validation
+On your IIS servers, first resync the WMI counters.
 
-When you run `datadog-agent info` you should see something like the following:
+On Windows <= 2003 (or equivalent), run the following in cmd.exe:
 
-    Checks
-    ======
+```
+C:/> winmgmt /clearadap
+C:/> winmgmt /resyncperf
+```
 
-        iis
-        -----------
-          - instance #0 [OK]
-          - Collected 39 metrics, 0 events & 7 service checks
+On Windows >= 2008 (or equivalent), instead run:
 
-## Compatibility
+```
+C:/> winmgmt /resyncperf
+```
 
-The iis check is compatible with all major platforms
+## Connect the Agent \
+
+Create a file `iis.yaml` in the Agent's `conf.d` directory:
```

Comment:
Do we say `Create` everywhere? In terms of how I usually configure this by hand, I usually copy the `_.yaml.example` file and edit it.

This isn't wrong, I just found it odd in terms of my personal workflow.